src/pkg/manifests/package:pkg.p5m

```diff
@@ -23,16 +23,18 @@
 # Copyright 2020 OmniOS Community Edition (OmniOSce) Association.
 #
 
+#
+# Don't worry about the "from x.y.z" six imports since pkgdepend has some issues
+# with the python importer. Instead, we force a dependency on the six package.
+#
 <transform file path=usr/bin/ -> set pkg.depend.bypass-generate .*six.*>
+<transform file path=.*\.py$ -> add pkg.depend.bypass-generate .*six.*>
 set name=pkg.fmri value=pkg:/package/pkg@$(PKGVERS)
 set name=pkg.summary value="Image Packaging System"
 set name=pkg.description \
     value="The Image Packaging System (IPS), or pkg(5), is the software delivery system used on OpenIndiana.  This package contains the core command-line components and pkg.depotd server."
 set name=info.classification \
     value=org.opensolaris.category.2008:System/Packaging
-set name=org.opensolaris.smf.fmri \
-    value=svc:/application/pkg/repositories-setup \
-    value=svc:/application/pkg/repositories-setup:default
```

Comment:
This is required for SMF to automatically refresh the service after upgrades. Mostly this is about mime database changes. I think this is not needed here. But just want to make sure we remove it on purpose.

Reply:
```
set name=org.opensolaris.smf.fmri 
  + value=svc:/application/pkg/dynamic-mirror value=svc:/application/pkg/dynamic-mirror:default value=svc:/application/pkg/mirror value=svc:/application/pkg/mirror:default value=svc:/application/pkg/server value=svc:/application/pkg/server:default
set name=pkg.fmri 
  - value=pkg://pkg5-nightly/package/pkg@0.5.11,5.11-0.151.1.8.1:20220128T222822Z
  + value=pkg://pkg5-nightly/package/pkg@0.5.11,5.11-0.151.1.8.1:20220128T222943Z
```

Better, not worse